### PR TITLE
Add EVO 4.1.4

### PR DESCRIPTION
**Pull request by colinrymer**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/evo.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "evo" do
+  version "4.1.4"
+  sha256 "894a6fdc75a05adc68911809dfb0d216d55d00ac511993a57beba44a6368b3dd"
+
+  # d9w4fhj63j193.cloudfront.net/EVO/EVO%208/Drivers/ was verified as official when first introduced to the cask
+  url "https://d9w4fhj63j193.cloudfront.net/EVO/EVO%208/Drivers/EVO%20v#{version}.dmg"
+  name "Evo"
```
Comment (vitorgalvao) [inline Casks/evo.rb:7]: `Audient Evo`

File: Casks/evo.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  desc "Audient Evo Control and Loop-back Mixer"
```
Comment (vitorgalvao) [inline Casks/evo.rb:8]: `Control and Loop-back Mixer`

File: Casks/evo.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask "evo" do
+  version "4.1.4"
+  sha256 "894a6fdc75a05adc68911809dfb0d216d55d00ac511993a57beba44a6368b3dd"
+
+  # d9w4fhj63j193.cloudfront.net/EVO/EVO%208/Drivers/ was verified as official when first introduced to the cask
+  url "https://d9w4fhj63j193.cloudfront.net/EVO/EVO%208/Drivers/EVO%20v#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/evo.rb:6]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://evo.audio/products/evo-4/downloads/`

File: Casks/evo.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  name "Evo"
+  desc "Audient Evo Control and Loop-back Mixer"
+  homepage "https://evo.audio/products/evo-4/downloads/"
+
+  app "EVO.app"
```
Comment (vitorgalvao) [inline Casks/evo.rb:11]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

**Comment by colinrymer** [comment]
https://github.com/Homebrew/homebrew-cask-drivers/pull/1750 opened.

**Pull request closed without merging**